karma.conf.js

```diff
@@ -28,10 +28,10 @@ module.exports = function(config) {
       // Zone.js dependencies
       'node_modules/zone.js/dist/zone.js',
       'node_modules/zone.js/dist/long-stack-trace-zone.js',
+      'node_modules/zone.js/dist/proxy.js',
```

Comment:
Is this required?

Reply:
My previous commit fixed the async test cases. This is the same file that is on the master branch. Project was rebased as well but for some reason it still shows it as a change.